### PR TITLE
Update user module documentation

### LINKED ISSUE #59796: module user: no notes for parameter shell
##### SUMMARY
The documentation of the module *user* / parameter *shell* states:

https://github.com/ansible/ansible/blob/8edad83ae029d7838a6d6347576d2ddbcce12ad0/lib/ansible/modules/system/user.py#L79

But there are no (shell specific) notes.
##### ISSUE TYPE
- Documentation Report

##### COMPONENT NAME
user / docs

##### ANSIBLE VERSION
devel branch

##### CONFIGURATION
N/A

##### OS / ENVIRONMENT
N/A

##### ADDITIONAL INFORMATION
Either remove the sentence or provide some information in the notes section.

### PR DESCRIPTION
**Pull request by ankitsagar**
##### SUMMARY
Updated the user module documentation as per @jkirk suggested
Fixes: #59796

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

user / docs

##### ADDITIONAL INFORMATION

Updated the documentation of the module user.

### CODE REVIEW

File: lib/ansible/modules/user.py [hunk 1]
```diff
@@ -70,8 +70,8 @@
             - Optionally set the user's shell.
             - On macOS, before Ansible 2.5, the default shell for non-system users was C(/usr/bin/false).
               Since Ansible 2.5, the default shell for non-system users on macOS is C(/bin/bash).
-            - On other operating systems, the default shell is determined by the underlying tool being
-              used. See Notes for details.
+            - On other operating systems, the default shell is determined by the underlying tool
```
Comment (Akasurde) [inline lib/ansible/modules/user.py:73]:
```suggestion
            - See notes for details how other operating systems determine the default shell by the underlying tool.
```

**Review by Akasurde — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/user.py [hunk 2]
```diff
@@ -70,8 +70,8 @@
...
+              (see Notes for details) being used.
```
Comment (Akasurde) [inline lib/ansible/modules/user.py]:
```suggestion
            - See notes for details on how other operating systems determine the default shell by the underlying tool.
```

**Review by Akasurde — CHANGES_REQUESTED** [review]
Reword the sentence.

File: lib/ansible/modules/user.py (hunk 2 above)
Comment (ankitsagar) [reply lib/ansible/modules/user.py]: @Akasurde I have done changes as you suggested

**Review by Akasurde — APPROVED** [review]

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
Thanks @ankitsagar for the Ansible documentation fix!

**Pull request merged**